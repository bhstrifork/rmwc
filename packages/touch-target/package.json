{
  "name": "@rmwc/touch-target",
  "version": "0.0.1",
  "main": "./index.js",
  "types": "./index.d.ts",
  "exports": {
    ".": {
      "import": "./index.mjs",
      "require": "./index.js"
    }
  },
  "sideEffects": [
    "./styles.tsx",
    "./styles.js",
    "*.css"
  ],
  "peerDependencies": {
    "@material/touch-target": "^14.0.0",
    "react": ">=16.8.x",
    "react-dom": ">=16.8.x"
  },
  "dependencies": {
    "@rmwc/base": "^14.0.1-alpha.0"
  }
}
