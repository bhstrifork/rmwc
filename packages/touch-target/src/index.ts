export * from './lib/touch-target';
