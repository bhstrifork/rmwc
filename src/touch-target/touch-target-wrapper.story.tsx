import React from 'react';

// import { storiesOf } from '@storybook/react';
import { TouchTargetWrapper } from '.';
import { Button } from '../button';

export default {
  title: 'TouchTargetWrapper',
  component: TouchTargetWrapper
};

// storiesOf('Buttons', module).add('Button', () => (
//   <TouchTargetWrapper>
//     <Button>Touchable</Button>
//   </TouchTargetWrapper>
// ));
