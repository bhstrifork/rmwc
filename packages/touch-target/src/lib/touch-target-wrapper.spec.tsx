import { render, screen } from '@testing-library/react';
import React from 'react';
import { TouchTargetWrapper } from './touch-target';

describe('TouchTargetWrapper', () => {
  it('renders', () => {
    const { asFragment } = render(
      <TouchTargetWrapper>TouchTargetWrapper</TouchTargetWrapper>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
