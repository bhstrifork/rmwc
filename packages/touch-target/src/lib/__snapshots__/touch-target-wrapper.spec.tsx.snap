// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`TouchTargetWrapper > renders 1`] = `
<DocumentFragment>
  <div
    class="mdc-touch-target-wrapper"
  >
    TouchTargetWrapper
  </div>
</DocumentFragment>
`;
